triveni-codev/GBWebApp-dev
# Darlp networks service extension 1203671430631939

Hi @triveni-codev ,

Kindly have someone to check this PR for ATSC 3 Service Extensions.

ASANA: https://app.asana.com/0/1201943683850147/1203671430631939

Considerations:
-this is limited only for atsc 3 types
-this is limited only for the first 3 service extension types (capabilities, drm system id and global service id)

To Test:
-adding and updating of atsc 3 service extensions on dirty change
-can retrieve any of these 3 existing service extension types from server, display and can be updated
-logics for the specified first 3 service extensions 
    >should allow any character inputs
    >should only have one instance (i.e if capabilities is selected, it shouldn't be available in the dropdown options) even on existing record from server

Do Not Test: Big Commit Add and Update
